src/pages/Editor.vue

```diff
@@ -194,9 +191,26 @@ export default {
         this.$refs.player.player.destroy();
       }
       this.videoId = linkValidation["ID"];
+      this.checkAndSavePlio();
```

Comment:
This is being called when watching `videoURL` and `plioTitle`.
Shouldn't it also be called when watching `itemTimestamps` and `items`?